CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to the Federal Front Door Microsite
+
+We aspire to create a welcoming environment for collaboration on this project.
+To that end, we follow the [18F Code of Conduct] [18f-coc] and ask that all
+contributors do the same.
+
+[18f-coc]: https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md "18F: Code of Conduct"
+
+ Section | Description
+ ------- | -----------
+ [Opening an Issue](#opening-an-issue) | Information on contributing Issues
+ [Submitting Pull Requests](#submitting-pull-requests) | Information on contributing Pull Requests
+ [Deployment Cycle](#deployment-cycle) | Information on the deployment cycle
+
+## Opening an Issue
+
+Before submitting a new Issue, please consider the following guidelines:
```

Comment:
So, I just checked out some of GitHub's documentation to find out more about the capitalization of "issue," and it looks like "Issues" (as a system) is capitalized, but "issue" (as in an individual issue) is lowercase. TL;DR — I think "issue" should be lowercase here.